site/docs/themes/extended-styling.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Extended styling
```

Comment:
I'm not sure the title reflects the page? Do we want to acknowledge that Salt components come with default tokens applied, however, these can be swapped for tokens of the same time (when they exist) that provide more or less affordance, e.g., borders in a card.